verification/curator-service/ui/src/components/CaseForm.tsx

```diff
@@ -202,7 +202,9 @@ const NewCaseValidation = Yup.object().shape(
             .max(120, 'Age must be between 0 and 120')
```

Comment:
at some point maybe we want to make 0 and 120 consts since they're referred to several times in here (and probably in bulk upload too)

Reply:
Yes good point, added TODO.